--- origsrc/shishensho-0.3.1/Makefile.am	2008-07-03 19:08:14.000000000 -0500
+++ src/shishensho-0.3.1/Makefile.am	2008-07-30 01:09:44.765625000 -0500
@@ -27,12 +27,12 @@
 
 install-data-local:
 	$(INSTALL) -d $(DESTDIR)/$(datadir)/applications/hildon ;
-	$(INSTALL) maemo/shishensho.desktop $(DESTDIR)/$(datadir)/applications/hildon/shishensho.desktop ;
+	$(INSTALL) $(srcdir)/maemo/shishensho.desktop $(DESTDIR)/$(datadir)/applications/hildon/shishensho.desktop ;
 	$(INSTALL) -d $(DESTDIR)/$(datadir)/$(PACKAGE) ;
-	$(INSTALL) glade/guigtk.glade $(DESTDIR)/$(datadir)/$(PACKAGE) ;	
+	$(INSTALL) $(srcdir)/glade/guigtk.glade $(DESTDIR)/$(datadir)/$(PACKAGE) ;	
 	$(INSTALL) -d $(DESTDIR)/$(datadir)/$(PACKAGE)/pixmaps ;
-	$(INSTALL) pixmaps/kshisen_bgnd.png $(DESTDIR)/$(datadir)/$(PACKAGE)/pixmaps ;	
-	$(INSTALL) pixmaps/tileset.png $(DESTDIR)/$(datadir)/$(PACKAGE)/pixmaps ;	
-	$(INSTALL) pixmaps/tileset_letters.png $(DESTDIR)/$(datadir)/$(PACKAGE)/pixmaps ;	
+	$(INSTALL) $(srcdir)/pixmaps/kshisen_bgnd.png $(DESTDIR)/$(datadir)/$(PACKAGE)/pixmaps ;	
+	$(INSTALL) $(srcdir)/pixmaps/tileset.png $(DESTDIR)/$(datadir)/$(PACKAGE)/pixmaps ;	
+	$(INSTALL) $(srcdir)/pixmaps/tileset_letters.png $(DESTDIR)/$(datadir)/$(PACKAGE)/pixmaps ;	
 
 ## End Makefile.am
